### ISSUE TITLE
Sort order in the metabox lost or reversed

### ISSUE THREAD

**Issue by peroks-reader** [wpmetabox/mb-relationships#13]
Hi,

in the MB Relationships ADMIN “from”-Metabox, the sort order of the related “to”-posts gets lost whenever a “to”-post is edited and saved. Sometimes the sort order gets reversed.

So whenever I have edited and saved a “to”-post and later want to edit a “from”-post, I have to manually reorder all “to”-posts in the MB Relationships Metabox before saving. This is slightly annoying in development and a no-go in production.

Can you please look into this?
Thanks
Per

Here is my MB Relationships config:

```
MB_Relationships_API::register( array(
‘id’ => ‘tours_to_locations’,
‘from’ => array(
‘object_type’ => ‘post’,
‘post_type’ => ‘product’,
‘meta_box’ => array( ‘context’ => ‘advanced’, ‘title’ => ‘Locations’ ),
),
‘to’ => array(
‘object_type’ => ‘post’,
‘post_type’ => ‘location’,
‘admin_column’ => ‘after title’,
‘meta_box’ => array( ‘context’ => ‘normal’, ‘title’ => ‘Used in Tours’ ),
),
) );
```

**Comment by aripekkako** [comment]
We are facing this same issue. It known by the developers: https://metabox.io/support/topic/drag-and-drop-order-reset-without-consent/

**Comment by rilwis** [comment]
We're fixing the bug. It might require database structure change. So it's kind of a big deal.

Please wait.

**Comment by rilwis** [comment]
This is fixed in version 1.5.0. Please upgrade.
